flask_mongorest/operators.py

```diff
@@ -14,29 +17,35 @@ def __call__(self):
 
     def prepare_queryset_kwargs(self, field, value, negate):
         if negate:
-            return {'__'.join(filter(None, [field, 'not', self.op])): value}
+            return {'__'.join([_f for _f in [field, 'not', self.op] if _f]): value}
```

Comment:
`filter` still exists in Python 3.5.2, why change? https://docs.python.org/3/library/functions.html#filter